api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.0.x.xml

```diff
@@ -1790,6 +1790,7 @@
 		<validCheckSum>1b6141be59a8a243b9e56d0aad952af</validCheckSum> <!-- checksum from before idExceptions added -->
 		<validCheckSum>89cc7a14b0582f157ea2dbb9de092fd</validCheckSum> <!-- current checksum with idExceptions param added -->
 		<validCheckSum><comment>Changes to new concept_reference tables</comment>3:4edd135921eb263d4811cf1c22ef4846</validCheckSum>
+		<validCheckSum><comment>Account for trailing whitespace</comment>9:5d41399f930d6de5c0030986b78f35e5</validCheckSum>
```

Comment:
Was this brought about by the liquibase upgrade?

Reply:
Yes IIRC there were some changes to how whitespace was handled so there were a few checksum mismatches